src/ceph.in

```diff
@@ -1005,6 +1006,16 @@ def main():
         print('error handling command target: {0}'.format(e), file=sys.stderr)
         return 1
 
+    # ceph iostat
+    if len(childargs) > 0 and childargs[0] == 'iostat':
+        def call_iostat():
+            while 1:
+                subprocess.call([sys.argv[0], 'mgr', 'iostat'])
```

Comment:
As of now, it doesn't seem like there's a way to keep an open connection between a mgr plugin and the client. The alternative, which is implemented here, is having the client invoking the `iostat` plugin periodically.

Reply:
@badone added a blocking mode in `scrub` command. https://github.com/ceph/ceph/pull/19793
 maybe we could do this in same way.

Reply:
@Liuchang0812 Thanks for the comments. The issue is that (to my knowleddge) there's no way for the plugin to continuously communicate with the client. Adding a blocking mode for the client wouldn't solve that.